app/services/PaypalService.scala

```diff
@@ -50,7 +49,8 @@ class PaypalService(
   config: PaypalApiConfig,
   contributionData: ContributionData,
   identityService: IdentityService,
-  emailService: EmailService
+  emailService: EmailService,
+  supportPaypalExecuteEndpoint: String
```

Comment:
I guess it would be nice if you could just pass in the baseReturnUrl, rather than have the support endpoint passed in but the contribute endpoint fetched from config within the `PaypalService`. Though having looked at how baffling the config is ([this line](https://github.com/guardian/contributions-frontend/blob/34bef65b8825c2b413c18857d9001b9db2c4a8f8/test/resources/application.conf#L41-L42) in particular had me scratching my head... do we even use Jenkins!??) I can get why didn't want to change any of this. And this codebase is going away soon...

Reply:
Yep I'm gonna leave this one for now.